apps/www/components/Frame/index.js

```diff
@@ -128,60 +131,67 @@ const Frame = ({
   }, [hasSecondaryNav])
   return (
     <ColorContextProvider colorSchemeKey={pageColorSchemeKey}>
-      <ColorHtmlBodyColors colorSchemeKey={pageColorSchemeKey || 'auto'} />
-      <noscript>
-        <Box style={{ padding: 30 }}>
-          <RawHtml
-            dangerouslySetInnerHTML={{
-              __html: t('noscript'),
-            }}
-          />
-        </Box>
-      </noscript>
-      <div
-        {...(footer || inNativeApp ? styles.bodyGrowerContainer : undefined)}
+      <ColorHtmlBodyColors
+        colorsObject={isClimate ? colors.climateColors : colors}
```

Comment:
```suggestion
        colorsObject={isClimate ? colors.climateColors : undefined}
```
The ColorHtmlBodyColors has colors as a fallback if undefined is provided, so we don't have to provide it by ourselves here again.